### ISSUE TITLE
ptpSyncInterval parameter name and default value

### ISSUE THREAD

**Issue by eduardoyusta** [openBackhaul/synchronization#36]
Taking as a reference the latest shared gendoc file (v.220404.1830_AlWe_TH), the description indicates the parameter represents the exponent of a 2^N number.

IEEE 1588-2008 & 2019 specify the syncInterval= 2^(logSyncInterval), being logSyncInterval an INT8 value (e.g 1588-2019 sec. 8.2.15.4.3).
ITU profile 8275.1 includes (table A.5) as part of its specification the logSyncInterval parameter as well.

This way, if the intended use of the parameter is as the exponent to set the 2^N syncInterval, probably makes sense to rename this parameter to ptpLogSyncInterval to keep consistency with specifications. Current type defined for the parameter is already INT8.

As for the default value, although defult profile in IEEE specs (e.g section I.3.2 1588-2019) specifies inicialization to 0 and range -1..1, ITU 8275.1 (profile of interest for actual implementation) sets -4 as fixed value for ptpLogSyncInterval (inicialization and range for BC/TC/OC). Consider whether to keep current -1 value as default or modify, if more differentiation from the main working value is desired.

**Comment by DanielaSpreafico** [comment]
I agree to rename this attribute to  ptpLogSyncInterval in the range (-7..-4)

**Comment by openBackhaul** [comment]
**Proposal to the 5G-xhaul call on 7th of September 2022:**
Name of ptpSyncInterval shall be changed to ptpLogSyncInterval .
Datatype shall be changed from INT16 to INT8.
Default value shall be changed from -1 to -99.

**Comment by Nzein** [comment]
If the telecom profile to be used is known, it is safer to specify the default values defined in the telecom profile.

**Comment by demx8as6** [comment]
> If the telecom profile to be used is known, it is safer to specify the default values defined in the telecom profile.

If a device is using the telecom profile from ITU-T then the parameters in LTP::*_PAC are set accordantly. 
However, when a certain parameter is not implemented, then the default value out side the "valid" range (valid: depending on the profile) - in this case -99 - should be reported by the device.

**Comment by demx8as6** [comment]
Decision made at the 5G-xhaul call on 7th of September 2022:
Name of ptpSyncInterval shall be changed to ptpLogSyncInterval .
Datatype shall be changed from INT16 to INT8.
Default value shall be changed from -1 to -99.
